vowpalwabbit/metrics.cc

```diff
@@ -70,6 +70,8 @@ void list_to_json_file(dsjson_metrics* ds_metrics, std::string filename, metric_
       writer.String(ds_metrics->LastEventId.c_str());
       writer.Key("last_event_time");
       writer.String(ds_metrics->LastEventTime.c_str());
+      writer.Key("cbea_sum_cost_original");
```

Comment:
cbea is the prefix used by cb_explore_adf, in this case this value comes from dsjson, I would rename to dsjon_sum_cost_original?